packages/data-controls/README.md

```diff
@@ -74,7 +74,7 @@ resolvers,
 
 _Parameters_
 
--   _paths_ (unknown type): 
+-   _paths_ ``: 
```

Comment:
It still isn't working.

Reply:
It is missing the type: https://github.com/WordPress/gutenberg/blob/d9e7541fe29c0022f2f5de2570ec60bee670aede/packages/data-controls/src/index.js#L106

Reply:
Can we remove the `@param paths` from here? I'm not familiar with this part of the code but the param doesn't make sense as it is.